app/DashboardWidgetWrapper.js

```diff
@@ -18,58 +20,88 @@ export const DashboardContainer = styled.div`
 `;
 
 /**
- * Returns the ContentAnalysis component.
+ * Returns the DashboardWidget component.
  *
- * @returns {ReactElement} The ContentAnalysis component.
+ * @returns {ReactElement} The DashboardWidget component.
  */
-export default function DashboardWidget() {
-	const seoAssessmentItems = [
-		{
-			value: 33,
-			color: "#F00",
-			html: "Posts with a <b>bad</b> score",
-		},
-		{
-			value: 20,
-			color: "#FF0",
-			html: "Posts with a <b>decent</b> score",
-		},
-		{
-			value: 47,
-			color: "#0F0",
-			html: "Posts with a <b>good</b> score",
-		},
-	];
+export default class DashboardWidget extends React.Component {
+	/**
+	 * Creates the components and initializes its state.
+	 */
+	constructor() {
+		super();
+
+		this.state = {
+			seoAssessmentItems: [
+				{
+					value: 33,
+					color: "#F00",
+					html: "Posts with a <b>bad</b> score",
+				},
+				{
+					value: 20,
+					color: "#FF0",
+					html: "Posts with a <b>decent</b> score",
+				},
+				{
+					value: 47,
+					color: "#0F0",
+					html: "Posts with a <b>good</b> score",
+				},
+			],
+			feed: null,
+		};
+
+		this.getFeed( "free" );
+	}
+
+	/**
+	 * Fetches data from the yoast.com feed, parses it and sets it to the state.
+	 *
+	 * @returns {void}
+	 */
+	getFeed() {
+		// Developer note: this link should -not- be converted to a shortlink.
+		getPostFeed( "https://yoast.com/feed/widget/", 3 )
+			.then( ( feed ) => {
+				feed.title = "Feed title";
+				feed.link  = "https://www.yoast.com";
+
+				feed.items = feed.items.map( ( item ) => {
+					item.description = item.description;
+					item.description = item.description.replace( `The post ${ item.title } appeared first on Yoast.`, "" ).trim();
+					item.content = item.content;
+
+					return item;
+				} );
+
+				this.setState( { feed } );
+			} )
+			/* eslint-disable-next-line no-console */
+			.catch( error => console.log( error ) );
```

Comment:
Probably better to make this a `console.error`.

Reply:
I'm not sure we should throw a console.error in the users browser. More importantly, the general usage of `console` in the yoast-components should be re-thought, as it's one of the reasons why the standalone app doesn't work in Internet Explorer 11. In fact, in IE11, the console object is not available by default. It is only when the "dev tools" are open. So, with the dev tools closed, all you can see is a blank page.

If it was up to me, I'd opt for removing any usage of `console` from our codebase. Also, there's a reason why ESLint has a `no-console` rule and I'm not sure disabling the rule is a good path forward.

Reply:
Whether it's a good idea to use the console at all is another discussion. For now, let's change it to console.error, because it stands out more and might signal a bigger problem in the feed, and therefore in the plugin.